src/Tools/SarifValidator/Options.cs

```diff
@@ -20,5 +20,12 @@ internal class Options
             HelpText = "Path of the SARIF file to validate",
             Required = true)]
         public string InstanceFilePath { get; set; }
+
+        [Option(
+            'l',
+            "log-file-path",
+            HelpText = "Path to SARIF log file containing the results of the validation",
+            Default = "sarif.log")]
```

Comment:
Yes. You can add information on default output name conventions in the help attribute if you care to.